src/scripts/components/MultiAvatar.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class MultiAvatar extends React.Component {
+  renderInitials(initials) {
+    return (
+      <text x="50" y="50" textAnchor="middle" dy="0.35em" fontSize="30">{initials}</text>
+    );
+  }
+
+  formatInitials(name) {
+    const splitName = name.split(' ');
+    return splitName[0][0] + splitName[splitName.length - 1][0];
+  }
+
+  renderAvatars() {
+    const { foregroundImageUrl, backgroundImageUrl, foregroundName, backgroundName } = this.props;
+
+    const foregroundInitials = this.formatInitials(foregroundName);
+    const backgroundInitials = this.formatInitials(backgroundName);
+
+    const foregroundImage = foregroundImageUrl;
+    const backgroundImage = backgroundImageUrl;
+
+    return (
+      <div className="multi-avatars">
+        <svg
+          viewBox="0 0 100 100"
+          xmlns="http://www.w3.org/2000/svg"
+          preserveAspectRatio="none"
+          className="avatar avatar--member multi-avatars__background-avatar"
+          style={backgroundImage ? { backgroundImage: `url(${backgroundImage})` } : {}}
+        >
+          {backgroundImage === '' && this.renderInitials(backgroundInitials)}
```

Comment:
I think for this check to work we need to add default prop values to the background and foreground **image** props, since they will be undefined if they are not passed down, so basically with the logic above if you don't pass a image URL, it will also NOT render the initials, so you'll end up with a blank avatar.

Reply:
🔥